index.js

```diff
@@ -1 +1,120 @@
-console.log("hello world!");
+let ORIGINAL_BODY_OVERFLOW;
+let ORIGINAL_BODY_PADDING_RIGHT;
+
+// elements
+const openButtonEl = document.getElementById("open-button");
+const closeButtonEl = document.getElementById("close-button");
+const dialogEl = document.getElementById("dialog");
+const dialogContentEl = document.getElementById("dialog-content");
+
+// lock/unlock scroll
+function lockBody() {
+  // Save initial body style
+  ORIGINAL_BODY_OVERFLOW = document.body.style.overflow;
+  ORIGINAL_BODY_PADDING_RIGHT = document.body.style.paddingRight;
+
+  // Lock body scroll
+  document.body.style.overflow = "hidden";
+
+  // Get the scrollBar width
+  const root = document.body;
+  const scrollBarWidth = root.offsetWidth - root.scrollWidth;
+
+  // Avoid width reflow
+  if (!scrollBarWidth) {
```

Comment:
Неявне приведення типів це верогідна помилка. Може краще 
```suggestion
  if (!scrollBarWidth <= 0) {
```